template.handlebars

```diff
@@ -1,23 +1,23 @@
-{%- if locals.user -%}
-  {# if a user is logged in, we display either the view-mode or edit-mode toolbar, and associated scripts, styles, and templates #}
-  <div data-uri="{{ _ref or _self }}" class="clay-kiln">
+{{#if locals.user}}
```

Comment:
question: why is `@root` not used here? is it because this template is IN the root?